Deploy/Documentation/Notes/docModelNotes.json

```diff
@@ -24,6 +24,12 @@
           "%   |-------------- sub-01_TB1map.nii.gz",
           "%   |-------------- sub-01_TB1map.json",
           "% ",
+          "% <https://osf.io/rt642/download Download> our example |TB1DAM| BIDS dataset and process:",
```

Comment:
BIDS demo documentation, making a note to myself to test this manually.

Reply:
Seems like it gets repeated for each model, so also making a note to check that this automated approach at generating the BIDS documentation is general enough for all models that it's generated for.

Reply:
The notes field is model specific to add BIDS-specific content and citation strings to the reference articles: 

![image](https://user-images.githubusercontent.com/9632322/236508548-5b45bfee-f429-4547-ab64-3752785bad5f.png)

Handled by the following https://github.com/qMRLab/qMRLab/blob/af5c37f8e8ddbc21a786b6bd5b0d37a4a69f8511/src/Common/qMRgenBatch.m#L196-L221